### PR TITLE
Add option to indent code after label

### PR DESCRIPTION
**Pull request by kyklish**
Changes proposed in this pull request:

- new setting to disable indentation after label:

```autohotkey
;On
label:
    MsgBox
return

;Off
label:
MsgBox
return
```

Notifying @mark-wiemer

### CODE REVIEW

**Comment by kyklish** [comment]
I don't know how to mock extension's settings in src/test/suite/format/format.test.ts, so I can't write test for my new setting :(

**Review by mark-wiemer — CHANGES_REQUESTED** [review]
Let's add some tests both with and without this setting enabled, just to be safe :)

**Comment by kyklish** [comment]
Done.

File: src/test/suite/format/format.test.ts
Comment (kyklish) [inline src/test/suite/format/format.test.ts:92]: Do we need save previous config value if we test it? Maybe set non-default value to test and then remove it from configuration and VSCode will set it to default value?

**Comment by mark-wiemer** [comment]
Hey @kyklish , I'll review this once merge conflicts are resolved :)

**Comment by kyklish** [comment]
Done.

File: src/test/suite/format/format.test.ts
Comment (mark-wiemer) [reply src/test/suite/format/format.test.ts:97]: Sorry for the delay. I understand your concern here. See #237 for a similar concern and #239 for a solution. Let me work on that PR so we can unblock this one :)

Comment (mark-wiemer) [reply src/test/suite/format/format.test.ts:97]: @kyklish try now :)

**Comment by kyklish** [comment]
> Let's add some tests both with and without this setting enabled, just to be safe :)

Done, ready for review.

**Review by mark-wiemer — APPROVED** [review]

**Pull request merged**
